responsibleai/responsibleai/_tools/causal/dashboard_schemas/schema_0.1.0.json

```diff
@@ -0,0 +1,220 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
will we be removing this schema from azureml-responsibleai package now?  The schema checks will always be done in OSS now?

Reply:
Yep! Currently working on another branch to remove it